browser-test/src/support/admin_programs.ts

```diff
@@ -857,4 +857,12 @@ export class AdminPrograms {
 
     await this.publishProgram(programName)
   }
+
+  getInputByName(name: string) {
+    return this.page.locator(`input[name=${name}]`)
+  }
+
+  async clickButtonWithText(text: string) {
+    await this.page.click(`:is(button:has-text("${text}"))`)
+  }
```

Comment:
suggestion (blocking): these are both overly generic methods for where they are. Recommend inlining the arguments passed to them so they have behavior specific to admin programs (e.g. `getInputByName(name: string)` -> `getAddressCorrectionToggle()`) or moving them to a more appropriate file.

Reply:
I renamed the two helper methods per your suggestion.